docs/webhooks/Secrets.md

```diff
@@ -50,8 +52,8 @@ timestamp = signatureSplitByComma[0].Split("t=")[1]
 signatureHmac = signatureSplitByComma[1].Split("v1=")[1]
 
 content = concatenate(body, timestamp)
-contentHmac = getHmacSha256(content, secretDecoded)
```

Comment:
Why has this been removed?

Reply:
that's weird, I'm not sure what happened there. Added it back